### ISSUE TITLE
Infill direction changes Top Surface / Skin extrusion direction

### ISSUE THREAD

**Issue by Ltek** [SoftFever/OrcaSlicer#2272]
### OrcaSlicer Version
1.7
1.8
1.9

### Printer
ALL

### How to reproduce

Change Advanced -> Infill Direction (see highlight in screenshot)

![Capture](https://github.com/SoftFever/OrcaSlicer/assets/1626802/5b5a28c7-bc84-4163-bce9-3598c1f676ed)

### Actual results

The top layer extrusion direction will change based on the value set for Infill Direction. 

### Expected results

Top layer extrusion direction should not change based on the value set for Infill Direction. I'm also filing a Feature Request to 'Force Top Layer Skin extrusion direction" which is useful for designs that want/use the extrusion lines in the final design look.

### Project file & Debug log uploads

not necessary, happens with every/any project

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by Ltek** [comment]
hello

**Comment by Ltek** [comment]
still a problem

**Comment by Ltek** [comment]
not stale... @SoftFever any chance you could add this? Cura has had this for years

**Comment by Ltek** [comment]
still need this. does any dev even look at these FRs? been over a year and still not a mention of this from a dev... wow

**Comment by SoftFever** [comment]
Can you post some use cases of such settings with preferable photos?

**Comment by Ltek** [comment]
> Can you post some use cases of such settings with preferable photos?

the issue is that we cannot control the Top Surface / Skin direction so this setting, which is under Strength -> Advanced -> Infill Direction influences the top layer's direction. A 'strength' feature should be about strength, as most would assume. 

But if anyone wants to control the top surface directly for cosmetic reasons, there is no way to do that.  If you have a multi-level top-surface and you want all the layer lines to be horizontal, that is somewhat impossible. Or only will happen is you are 'lucky'

While this is not the best example it does show the problem., there is no possible way it seems (I've spend hours testing different settings, posting on Reddit, etc - no one knows how to do it) to get all the top surface extrusions the same direction. And when you change this Infill Direction, it changes the top layer's direction too, not just the "infill" (internal sparse or solid)
<img width="704" alt="image" src="https://github.com/user-attachments/assets/bc171743-0cdc-4530-b45e-d6bba0367f72">

**Comment by SoftFever** [comment]
> > Can you post some use cases of such settings with preferable photos?
> 
> the issue is that we cannot control the Top Surface / Skin direction so this setting, which is under Strength -> Advanced -> Infill Direction influences the top layer's direction. A 'strength' feature should be about strength, as most would assume.
> 
> But if anyone wants to control the top surface directly for cosmetic reasons, there is no way to do that. If you have a multi-level top-surface and you want all the layer lines to be horizontal, that is somewhat impossible. Or only will happen is you are 'lucky'
> 
> While this is not the best example it does show the problem., there is no possible way it seems (I've spend hours testing different settings, posting on Reddit, etc - no one knows how to do it) to get all the top surface extrusions the same direction. And when you change this Infill Direction, it changes the top layer's direction too, not just the "infill" (internal sparse or solid) <img alt="image" width="704" src="https://private-user-images.githubusercontent.com/1626802/374391467-bc171743-0cdc-4530-b45e-d6bba0367f72.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0MjgzMzgsIm5iZiI6MTcyODQyODAzOCwicGF0aCI6Ii8xNjI2ODAyLzM3NDM5MTQ2Ny1iYzE3MTc0My0wY2RjLTQ1MzAtYjQ1ZS1kNmJiYTAzNjdmNzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDhUMjI1MzU4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Nzg2NDEwMTFkODBhYmJlNTExMTJjOGIyNjQwNjE0YjcyNGI0NjFlYjI3ZmQyYjRlNjY0OTg0Y2E2YzZmOTA4ZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.rAmJsOIJqf9FR1ZkHuRyVFCiO0BraLN1Uz6ZCilWe6c">

Let me rephrase my question: in what situations do you think the top surface infill direction matters, and why? Do you have any photos of real prints to demonstrate the difference?

**Comment by Ltek** [comment]
> Let me rephrase my question: in what situations do you think the top surface infill direction matters, and why? Do you have any photos of real prints to demonstrate the difference?

Some large flat surfaces show the extrusion lines sot its ascetics. Hard to capture photos. Even with Matte black PLA which I use often, if you change directions of the lines it no longer looks as good... and can be very noticeable

Cura has had top and bottom layer direction for years...

<img width="250" alt="image" src="https://github.com/user-attachments/assets/4dae5722-70c6-4975-8367-54d0d08b41dd">
